### DISCUSSION TITLE
Microsoft Defender Application Guard Deprecated

### DISCUSSION THREAD

**Discussion by FowlFunction** [simeononsecurity/Windows-Optimize-Harden-Debloat#98] [Q&A] [answered]
According to the [article](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-security-windows-defender-application-guard) linked in the recommended reading section:
> Microsoft Defender Application Guard, including the [Windows Isolated App Launcher APIs](https://learn.microsoft.com/en-us/windows/win32/api/isolatedapplauncher), will be deprecated for Microsoft Edge for Business and [will no longer be updated](https://learn.microsoft.com/en-us/windows/whats-new/deprecated-features).

Are there any native alternatives you'd recommend? From a brief search it appears the closest replacement would be either windows sandbox or hyper-V. I'm looking for a long-term solution that provides browser isolation for edge.

[a comment by an automated account is not shown]

**Comment by simeononsecurity** [comment, marked as answer]
> According to the [article](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-security-windows-defender-application-guard) linked in the recommended reading section:
> 
> > Microsoft Defender Application Guard, including the [Windows Isolated App Launcher APIs](https://learn.microsoft.com/en-us/windows/win32/api/isolatedapplauncher), will be deprecated for Microsoft Edge for Business and [will no longer be updated](https://learn.microsoft.com/en-us/windows/whats-new/deprecated-features).
> 
> Are there any native alternatives you'd recommend? From a brief search it appears the closest replacement would be either windows sandbox or hyper-V. I'm looking for a long-term solution that provides browser isolation for edge.

All the solutions purposed are mostly just workarounds. They are only as secure as how you use them.
Easiest one to set up would be the windows sandbox. However the sandbox is basically just a vm running on your computer.
It has some benefits over just a normal vm. But in reality, you can probably just use vmware workstation or virtualbox and get the same level of separation. If you're in an enterprise, v-app would be another option.  It's really just up to you. IMO they are all terrible.
